eng/integration-tests.yml

```diff
@@ -37,15 +39,18 @@ steps:
       mkdir -p ~/bin
       export PATH="~/bin:$PATH"
       export GO111MODULE=on
+      cd '$(sdkPath)'
       make test-cover
-      goveralls -coverprofile=cover.out -service=azure-devops
       make destroy
+      gocov convert cover.out > coverage.json
+      gocov-xml < coverage.json > coverage.xml
+      gocov-html < coverage.json > coverage.html
     displayName: 'Run Integration Tests'
     env:
       ARM_SUBSCRIPTION_ID: $(go-live-azure-subscription-id)
       ARM_CLIENT_ID: $(go-live-eh-azure-client-id)
       ARM_CLIENT_SECRET: $(go-live-eh-azure-client-secret)
-      ARM_TENANT_ID: $(go-live-azure-tenant-id)
+      ARM_TENANT_ID: $(go-live-tenant-id)
```

Comment:
Is this the only one without a `azure-` in its name?

Reply:
Yes :( We could update it to have more consistency, but this will also affect SB.